mpqp_scripts/update_qiskit.py

```diff
@@ -0,0 +1,42 @@
+import subprocess
+import sys
+
+qiskit_packages = [
```

Comment:
_Bonus remark, we can address it later._
Do you think we can build this list dynamically for each user ? In case of our list is not exhaustive.
Something like ``pip list | grep qiskit`` and ``pip list | grep ibm``.

Reply:
I'd say we put this on trello

Reply:
Agreed, done. I resolve